pkg/pipeline/extract/conntrack/aggregator.go

```diff
@@ -109,10 +110,15 @@ func (agg *aggregateBase) getOutputField(d direction) string {
 func (agg *aggregateBase) getInputFieldValue(flowLog config.GenericMap) (float64, error) {
 	rawValue, ok := flowLog[agg.inputField]
 	if !ok {
-		if agg.metrics != nil {
-			agg.metrics.aggregatorErrors.WithLabelValues("MissingFieldError", agg.inputField).Inc()
+		// error only if explicitly specified as FLP skip empty fields by default to reduce storage size
+		if agg.reportMissing {
```

Comment:
I assume netobserv app always expect this bool to be `false` ? when can this bool be `true` I am wondering if we even need to bool and we can omit the error always ?

Reply:
I feel this could be usefull even for `netobserv`. 
`FlowDirection`, `IfDirection`, `TimeFlowStartMs`, `Bytes`, `Packets` fields for example.